dataeng/resources/tableau-config-backup.sh

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env bash
```

Comment:
Can you add a description at the top of this script indicating what it is used for and how to invoke it?

Reply:
Done it. thanks